src/main/resources/messages.properties

```diff
@@ -1,9 +1,11 @@
 # Validation messages for Entity fields
+assignment.not.exists = Doctor is not yet assigned to patient
 doctor.aadhaar.number.already.present = Aadhaar already present in database
 doctor.aadhaar.number.invalid.format = Aadhaar number must be exactly 12 digits and contain only numeric characters
 doctor.aadhaar.number.mandatory = Aadhaar number is mandatory
 doctor.aadhaar.number.should.not.be.empty = Aadhaar should not be empty!
 doctor.already.assigned = Doctor already assigned to Patient
+doctor.already.unassigned = Doctor already unassigned
```

Comment:
change this message Doctor already unassigned to 
The doctor has already been unassigned.